measures/2018/measures-schema.yaml

```diff
@@ -1,7 +1,8 @@
-id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/measures-schema.yaml
-$schema: http://json-schema.org/schema#
+$id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/measures-schema.yaml
+$schema: http://json-schema.org/draft-07/schema
```

Comment:
Curious what's this for? Cause I can't imagine it's making a network call to download the schema when verifying this file...?

Reply:
nope no effect, just for a reader

https://spacetelescope.github.io/understanding-json-schema/basics.html#declaring-a-json-schema
```
The $schema keyword is used to declare that something is JSON Schema. It’s generally good practice to include it, though it is not required.

You can also use this keyword to declare which version of the JSON Schema specification that the schema is written to.
```